packages/sdk-snippets/src/targets/ruby/rails/webhooks/fixtures/quirky-escaping.rb

```diff
@@ -0,0 +1,41 @@
+require 'readme/webhook'
+
+class MetricsController < ApplicationController
+  def index
+    render json: { message: 'hello world' }
+  end
+
+  def post
+    head :ok
+  end
+
+  def webhook
+    # Your ReadMe secret
+    secret = 'my-readme-secret'
+    # Verify the request is legitimate and came from ReadMe
+    signature = request.headers['readme-signature']
+
+    begin
+      Readme::Webhook.verify(request.raw_post, signature, secret)
+    rescue Readme::MissingSignatureError, Readme::ExpiredSignatureError, Readme::InvalidSignatureError => e
+      # Handle invalid requests
+      render json: { error: e.message }, status: 401
+      return
+    end
+
+    # Fetch the user from the database and return their data for use with OpenAPI variables.
+    # current_user ||= User.find(session[:user_id]) if session[:user_id]
+    render json: {
+      # OAS Server variables
+      '2name': 'default-name',
+      '*port': '',
+      'p*o?r*t': '',
+      normal_server_var: '',
+
+      # OAS Security variables
+      '"petstore" auth': 'default "key"\\',
```

Comment:
rad that this escaping work i did just works in ruby.

Reply:
Yeah it's wacky as hell but it seems to!

```
$ irb
irb(main):003:0> a = { '"petstore" auth': 'default "key"\\' }
=> {:"\"petstore\" auth"=>"default \"key\"\\"}
irb(main):004:0> a[:"\"petstore\" auth"]
=> "default \"key\"\\"
```